reports/blueprint/report.tex

```diff
@@ -255,22 +255,31 @@ \subsection{Milestones and Division of Labour}
         12 & Testing of complete system & Week 20 & All members \\
         13 & Open House & Week 22 & \\
         14 & Final deliverable & Week 23 & \\
-        15 & Final project report & Week 24 & All members \\
+        15 & Final project report & Week 24 & All members
     \end{tabularx} 
-\label{hw:specs-table}
+\label{hw:milestones-table}
 \end{table}
 \section{Budget} % Ethan
+% include some preamble wrt board quantities and BOM table
+% consider ActiveBOM from Altium for this.
+% BOM probably takes too many pages, will need to attenuate
+
 The budget of 600 dollars provided by the department is sufficient for this
 project assuming only one unit of the device is manufactured. In reality three
 PCB units are on order. However, the group will only submit a reimbursement
 request for one unit allowing the team to stay within budget. The extra two
 units will be treated as a personal expense since group members will be able to
 keep the extra devices and only submit the one paid for by the ECE department.
+Additional electrical instruments, such as oscilloscopes and signal generators
+for debugging are not included in the budget as they will be accessible via
+Walter Light Hall on campus.
+
+\subsection{Materials and Supplies}
 The following table breaks down the cost of the PCB fabrication and assembly as
-quoted by the supplier. The supplier contracted for TeachEE is Bittele (7PCB),
-they take care of both the fabrication and soldering of the PCBs. Moreover,
-Bittele takes the generated output from the EDA software and procures all the
-parts directly to their facility.
+quoted by the supplier. The supplier contracted for TeachEE is Bittele (7PCB).
```

Comment:
What is 7PCB?

Reply:
That is the other name the vendor goes by